modules/function/src/main/java/com/opengamma/strata/function/calculation/swaption/SwaptionCalculationMarketView.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (C) 2016 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.function.calculation.swaption;
+
+import com.opengamma.strata.calc.marketdata.CalculationMarketData;
+
+/**
+ * Market data for swaptions, used for calculation across multiple scenarios.
+ * <p>
+ * This interface exposes the market data necessary for pricing a swaption.
+ * This includes volatilities and rates.
+ * <p>
+ * Implementations of this interface must be immutable.
+ */
+public interface SwaptionCalculationMarketView {
```

Comment:
Happy with Scenario, I find just DataView a bit vague (its all data), and MarketDataView too long.